### ISSUE TITLE
error[E0554] during cargo build

### ISSUE THREAD

**Issue by oren** [twilco/kosmonaut#7]
```
cargo build

...
...
...

error[E0554]: `#![feature]` may not be used on the stable release channel
 --> src/main.rs:1:1
  |
1 | #![feature(or_patterns)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
```

More info:
```
rustc -V

rustc 1.45.2 (d3fb005a3 2020-07-31)
```

```
lsb_release -a

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
```

```
uname -a

Linux oren 4.4.0-179-generic #209-Ubuntu SMP Fri Apr 24 17:48:44 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by twilco** [comment]
Hi @oren!  Sorry for the hiccup -- Kosmonaut currently requires the nightly compiler.  Run `rustup default nightly` to make the switch.  We could probably get by without nightly, but the feature in your post makes the code quite a bit cleaner in some spots.

**Comment by Abendstolz** [comment]
@twilco What about adding a `rust-toolchain` file instead? That way people can stay on stable globally, but use nightly for this project. You could even pin the project to a specific `nightly` build for the time being.

https://github.com/rust-lang/rustup#the-toolchain-file

**Comment by twilco** [comment]
That's a great idea!  I'll do that now.

**Comment by twilco** [comment]
Fixed with aed29c5646d7bda5c0f1e7eb8c2b3992389d35ac.  @oren, if you have any more issues building the project feel free to open another issue.  Thanks!
